src/renderers/index.ts

```diff
@@ -146,7 +144,7 @@ class JavascriptRenderer implements IRenderer<Widget> {
     let w = new Widget();
     let s = document.createElement('script');
     s.type = mimetype;
-    s.textContent = data;
+    s.appendChild(document.createTextNode(data));
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) says "Setting this property on a node removes all of its children and replaces them with a single text node with the given value." The `textContent` syntax seems simpler, so I'd prefer reverting.